exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -86,6 +88,14 @@ type AzureManagedControlPlaneStatus struct {
 	// In the AzureManagedControlPlane implementation, these are identical.
 	// +optional
 	Initialized bool `json:"initialized,omitempty"`
+
+	// VirtualNetwork is the name of the AKS-generated VNET. It's not predictable unless we pre-create it.
+	// Consider pre-creating?
```

Comment:
"Consider pre-creating?" <-- Would you consider removing this and tracking it with an issue?

Reply:
Another way is to make AKS allow us to specify a vnet name

![](https://media.giphy.com/media/a5viI92PAF89q/source.gif)

Reply:
Agreed this is a little annoying in the AKS API right now, I’d likely prefer to extract it from agent pools for the sake of expediency but happy to ask about/volunteer to implement this in RP if that’s the path we want to go.

Reply:
yeah definitely meant as an alternative for that follow up issue, for now extracting the name is good